app/controllers/campaigns_controller.rb

```diff
@@ -134,7 +138,8 @@ def require_write_permissions
   end
 
   def set_campaign
-    @campaign = Campaign.find_by(slug: params[:slug])
+    return if @campaign = Campaign.find_by(slug: params[:slug])
```

Comment:
Set the campaign separately from the conditional return. Doing assignment in a condition is a rubocop style violation, because of the potential confusions between `=` and `==`.